packages/metamask/package.json

```diff
@@ -60,7 +60,7 @@
     "typescript": "^5.2.2"
   },
   "dependencies": {
-    "@metamask/sdk": "^0.20.3",
+    "@metamask/sdk": "^0.20.5",
```

Comment:
For a new package to publish the version above this on line 3 of this file needs a bump to `2.1.2-alpha.1` which will result in an alpha version published of `@web3-onboard/metamask`
We will also want to add this new version update to `packages/demo/package.json` and run `yarn && yarn check-all` from the root of the project and ensure there are not any errors